app-misc/cal/cal-4.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal  will display a calendar for the current month \
+with the current day marked. By specifing  certain arguments, cal will \
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE=""
```

Comment:
No homepage?

Reply:
Need to check my local recipe (latest), thought I'd put it in ...